src/assets/sass/protocol/components/_newsletter-form.scss

```diff
@@ -49,16 +49,16 @@
     @include hidden;
 }
 
+
 // Form wrapper with image
 .mzp-c-newsletter {
     margin: 0 auto $spacing-lg;
     max-width: $content-sm;
-    padding: $spacing-lg 0;
 
     @media #{$mq-md} {
         & {
             @include clearfix;
-            padding: $spacing-lg;
+            padding: 0 $spacing-lg;
```

Comment:
Hmm... maybe this is a case where it only loses vertical spacing when it's at the top or bottom of a container. So I think it could use `:first-child` and `:last-child` and otherwise use the variables by default. I did 0 because in most instances it's in a container of its own, but that won't always be the case. And if it is, then it's both the first and last child and that would be covered by those other rules. I'll update.

Reply:
Oh, my brain missed that this was horizontal spacing not vertical. In which case the spacing is quite different from the grid so I think it's fine to leave it.

Reply:
But you did prompt me to rethink it, and in cases when this might NOT be in its own container I still think I want to change it.